### DISCUSSION TITLE
Compatibility Matrix for KEDA and Kubernetes

### DISCUSSION THREAD

**Discussion by vnagireddy-virsec** [kedacore/keda#3295] [Q&A / Need Help]
Wondering if there is a version compatibility matrix for `KEDA` and `Kubernetes`. What I have seen in the documentation is a note saying Kubernetes cluster version v1.16+ is required.  Do any of you know about such compatibility matrix?

**Comment by tomkerkhove** [comment]
You can find our support policy here: https://github.com/kedacore/governance/blob/main/SUPPORT.md#kubernetes-support
